### DISCUSSION TITLE
How I can define another Layout in Volt class based component?

### DISCUSSION THREAD

**Discussion by yacobii** [livewire/livewire#6424] [1. Help]
<?php
 
use Livewire\Volt\Component;
 
new class extends Component {

} ?>
 
<x-app-layout>

</x-app-layout>

Here, the component renders the x-app-layout and also components.layouts.app. and How I can just render layouts.app?

**Comment by joshhanley** [comment]
@mehdiyaq Please ask volt questions in the volt repo, as it's managed by the Laravel team, not us 🙂
